src/App.tsx

```diff
@@ -18,6 +20,9 @@ function App() {
           <Route path="/restaurants" element={<Restaurants />} />
           <Route path="/chefs" element={<Chefs />} />
         </Routes>
+        <div className="slider-restaurant-container">
```

Comment:
the div with classname slider-restaurant-container should be inside the CardSlider component